### ISSUE TITLE
Create template for component epics

### ISSUE THREAD

**Issue by EoinShaughnessy** [alphagov/govuk-frontend#2561]
## What

Create a GitHub template for epics that are about publishing a component.

## Why

Many of the acceptance criteria are the same for every component we publish. A template would save people from having to write them out every time we create a new epic.

## Who needs to work on this

Technical writer, developers, anyone else involved in releases?

## Who needs to review this

Anyone involved in releases

## Done when

- [x] location found for template
- [ ] technical writer drafts template
- [ ] teammates review template
- [ ] template published

**Comment by kellylee-gds** [comment]
Hey @EoinShaughnessy just spotted this card and wondered if this doc may come in handy - https://docs.google.com/document/d/1SL6rncT-UYmgvcg3Yu_4YoRFHoSt995zx2BRTAnhTkE/edit. It was some work we did a while back to define acceptance criteria for building components and could be used as a template for an issue

**Comment by EoinShaughnessy** [comment]
@kellylee-gds Amazing, thank you - supreme sleuthing! I'll take a look at that Google Doc this week, and use it for the next iteration of [its PR](https://github.com/alphagov/govuk-frontend/pull/2562).

**Comment by trang-erskine** [comment]
@EoinShaughnessy This will be very useful for the team. Thanks for proactively looking at it. 

A few more sources of information that might be useful are
- https://github.com/alphagov/govuk-design-system/issues/1706 epic for accordion
- https://github.com/alphagov/govuk-frontend/issues/1935 epic for notification banner, although it's not very complete
- https://github.com/alphagov/govuk-frontend/blob/main/docs/contributing/test-components-using-accessibility-acceptance-criteria.md

**Comment by EoinShaughnessy** [comment]
@trang-erskine Thanks!

**Comment by EoinShaughnessy** [comment]
Following discussion with @kellylee-gds, we're going to try creating 3 templates, adapted from Kelly's [Google Doc on components' acceptance criteria](https://docs.google.com/document/d/1SL6rncT-UYmgvcg3Yu_4YoRFHoSt995zx2BRTAnhTkE/edit). These templates will cover the acceptance criteria for:

- designing a component
- building a component
- publishing a component

**Comment by EoinShaughnessy** [comment]
If we end up creating 3 templates, should we house them in their own 'Component epics' subfolder within the ['ISSUE_TEMPLATE' parent folder](https://github.com/alphagov/govuk-frontend/tree/8f0c3c185d0021f6c736c10942e7c3081d210571/.github/ISSUE_TEMPLATE)?

**Comment by trang-erskine** [comment]
@EoinShaughnessy at the moment when we click 'new issue' we can choose one of the existing templates. if the 3 new templates are within a subfolder, do you know if that still works?

**Comment by EoinShaughnessy** [comment]
@trang-erskine That's a really good question. I'm guessing it wouldn't, but I don't know for sure.

@lfdebrux What do you reckon? :)

**Comment by EoinShaughnessy** [comment]
@lfdebrux Good to know, cheers for confirming!

**Comment by EoinShaughnessy** [comment]
Hi @vanitabarrett!

The plan till now was to put the templates in the [ISSUE_TEMPLATE folder](https://github.com/alphagov/govuk-frontend/tree/main/.github/ISSUE_TEMPLATE) in this repo. However, when reviewing with @kellylee-gds, we wondered whether that might make the list of templates too long?

There's also an [issue-template folder in our internal repo](https://github.com/alphagov/design-system-team-internal/tree/master/.github). Could that work as a location for these component-issue templates? Users won't have access to them, but maybe for the time being the templates will likely just be for internal use...

**Comment by kellylee-gds** [comment]
@vanitabarrett yeah I can't think of an obvious way to make those templates more noticeable  🤔 At the moment I link to the comms template from the [Kanban principles card](https://github.com/orgs/alphagov/projects/4#card-55032900) but that can easily be missed
